### DISCUSSION TITLE
[Bug] Fresh Install Broken

### DISCUSSION THREAD

**Discussion by pin3c0n3** [FreshRSS/FreshRSS#6405] [General]
### Describe the bug

I completed a fresh install and the UI looks like links to scripts/formatting is missing, see attached. 
![Screenshot 2024-04-30 204333](https://github.com/FreshRSS/FreshRSS/assets/12443147/5ad3a732-4b03-4d40-850c-e03b0e5f3ad9)

### To Reproduce

Followed steps here: https://github.com/FreshRSS/FreshRSS#installation 

The install passed all the checks and at the end went to a screen like the one posted.  If I log in, content just goes along the left side, looking nothing like the demo or other screenshots on the site. 

### FreshRSS version

1.23.1

### Environment information

- Database version: [e.g. Mysql 5.7, SQLite]  I've tried both
- PHP version: [e.g. PHP 8.1] 8.2.15
- Installation type: [e.g. Docker, Docker image source, git, Yunohost] manual install/traditional web install
-Web server type: [e.g. Apache, nginx] Apache
- Device: [e.g. iPhone13] PC
- OS: [e.g. Ubuntu 22.04, Win10, MacOS14] Win 11
- Browser: [e.g. Firefox 124] Firefox 

### Additional context

Installation check
PHP installation

Your PHP version (8.2.15) is compatible with FreshRSS.

You have the cURL library.

You have PDO and at least one of the supported drivers (pdo_mysql, pdo_sqlite, pdo_pgsql).

You have the required library for regular expressions (PCRE).

You have the required library for character type checking (ctype).

You have the fileinfo library.

You have the required library to browse the DOM.

You have the JSON extension.

You have the recommended mbstring library for Unicode.

You have the ZIP extension.
File installation

Permissions on the data directory are good.

Permissions on the cache directory are good.

Permissions on the users directory are good.

Permissions on the favicons directory are good.

Permissions on the tokens directory are good.

**Comment by math-GH** [comment]
Hi, welcome new FreshRSS user :)

I looks like that you have troubles with the base URL.
Please check that the URL points to the `./p` directory and the `/i/` directory (inside of `./p`) is public available. The CSS files are stored inside of `./p/i`

(p = public, i = index)

After that, check the `base_url` in `./data/config.php` and set it correctly.

Does it help?

**Comment by pin3c0n3** [comment]
I see - so my we host has a folder, we'll call "www" - since the directions were to put everything in the p folder public, I put the contents of p into "www" and put everything else  ../* 

What is the recommended approach for a situation like this?  I didn't want to put all the contents in "www" since it says not to make the rest of the folders public.
